### PR TITLE
fix README to eliminate 'error: install has been replaced with add'

### PR DESCRIPTION
**Pull request by breinero**
Ticket(s): FE-###

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@
 ## Setting up this Repo
 
 1. Clone the repository; e.g. `gh repo clone fauna/fauna-js` if you use the GitHub CLI
-2. Install dependencies via `yarn install`
+2. Install dependencies via `yarn add`
```
Comment (cleve-fauna) [inline README.md:19]: This should stay install.

**Review by ptpaterson — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (ptpaterson) [inline README.md:19]:
```suggestion
2. Install dependencies via `yarn install`
```

**Review by ptpaterson — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (breinero) [reply README.md]: Ok. I need to learn more about Yarn! :)

**Comment by cleve-fauna** [comment]
Hey @breinero the other change was fine!

**Pull request closed without merging**